src/js/dataEntryPanel.js

```diff
@@ -146,6 +134,15 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             var deCont = floe.chartAuthoring.dataEntryPanel.append(that.locate("dataEntries"), dataEntryContainerTemplate);
             that.events.createDataEntryField.fire(deCont);
         }
+
+        var resetButton = that.locate("resetButton");
+
+        resetButton.click(function(e) {
+            if($(this).is(":focus")) {
```

Comment:
Is there a reason to have this ```if``` condition, as well as the ```preventDefault``` at line 144?

Line 138 - 145 can be made declarative in the component defaults block:
```
listeners: {
    "onCreate.bindResetClick": {
    	"this": "{that}.dom.resetButton",
    	method: "click",
    	args: ["{that}.resetForm"]
    }
}
```